features/step_definitions/account_creation.rb

```diff
@@ -10,54 +10,61 @@
   expect(page).to have_content "Log in"
 end
 
-When(/^I click "(.*?)"$/) do |arg1|
-  pending # express the regexp above with the code you wish you had
+When(/^I click the "(.*?)" link$/) do |arg1|
+  click_link arg1 
```

Comment:
I'd change the `arg1` paramater to `target` or something, just to make it easier to read.